src/components/LayoutTree.tsx

```diff
@@ -268,6 +268,8 @@ const linearizeTree = (
 
   const thoughtId = head(path)
   const thought = getThoughtById(state, thoughtId)
+  // if the thought is suddenly undefined, we opt for a blank tree, see getThoughtById
+  if (!thought) return []
```

Comment:
Let's move this into the ternary [below](https://github.com/cybersemics/em/blob/1de83fc608b916d44f83caf39032d3f5442b7b01/src/components/LayoutTree.tsx#L275) since `value` is only used there. That will have a narrower impact.

Reply:
I've found this approach to be purest if we put it in ternary.
```typescript
  const children = contextViewActive
    ? thought ? getContextsSortedAndRanked(state, thought.value) : []
    : // context children should render the children of a specific Lexeme instance to avoid repeating the Lexeme.
      // See: contextId (above)
      getChildrenRanked(state, contextId || thoughtId)
      ```